fili-core/src/test/groovy/com/yahoo/bard/webservice/application/LoaderSpec.groovy

```diff
@@ -139,6 +140,7 @@ class LoaderSpec extends Specification {
         scheduler.shutdownNow()
     }
 
+    @Ignore
```

Comment:
Can we give these a comment as to _why_ we're ignoring them? (ie. the tests are timing-based and therefore painfully flakey) I _think_ a comment can be passed to this annotation as a string value parameter.